src/main/java/org/example/FullResponse.java

```diff
@@ -0,0 +1,50 @@
+package org.example;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FullResponse {
```

Comment:
`FullResponse` is not really meaningful. Maybe `CocktailResponse`. Also, you don't need the `PartialResponse`